README.md

```diff
@@ -457,6 +459,33 @@ func FunctionReconciler(c reconcilers.Config) *reconcilers.ResourceReconciler[*b
 ```
 [full source](https://github.com/projectriff/system/blob/4c3b75327bf99cc37b57ba14df4c65d21dc79d28/pkg/controllers/build/function_reconciler.go#L39-L51)
 
+#### Advice
+
+[`Advice`](https://pkg.go.dev/reconciler.io/runtime/reconcilers#Advice) is a sub reconciler for advising the lifecycle of another sub reconciler in an aspect oriented programming (AOP) style. `Before` is called before the reconciler and `After` afterward. `Around` is used between Before and After to have full control over how the reconciler is called, including suppressing the call, modifying the input or result, or calling the reconciler multiple times.
```

Comment:
**[suggestion, non-blocking]:** might be that "delegated" isn't the right word here

```suggestion
[`Advice`](https://pkg.go.dev/reconciler.io/runtime/reconcilers#Advice) is a sub reconciler for advising the lifecycle of another sub reconciler in an aspect oriented programming (AOP) style. `Before` is called before the delegated reconciler and `After` afterward. `Around` is used between Before and After to have full control over how the delegated reconciler is called, including suppressing the call, modifying the input or result, or calling the reconciler multiple times.
```